crates/dprint/src/environment/environment.rs

```diff
@@ -59,7 +62,7 @@ macro_rules! log_verbose {
         if $environment.is_verbose() {
             let mut text = String::from("[VERBOSE]: ");
             text.push_str(&format!($($arg)*));
-            $environment.log(&text);
+            $environment.log_verbose(&text);
         }
```

Comment:
this guard can/should potentially go into `log_verbose` now, in case the function is used directly and not through the macro. Or alternatively if there is a way to prohibit direct via visibility use that would be a possible solution as well.